Assets/Scripts/Battle/BattleGUIBuilder.cs

```diff
@@ -73,14 +79,30 @@ IEnumerator LoadingSpriteAsyncFromPath(string path)
     public void SetParameters(Wizard wizard)
     {
         battleGUIWIzardMainStats.healthPointsText.text = wizard.HealthPoints.ToString();
```

Comment:
Tutaj jeszcze taka prośba.
Takie przypisanie ustawi nam raz na stałe wartość dla parametru "Health" na UI. Przydałoby się dodać w Wizard eventy w stylu onHealthChanged() , onConditionChanged, onManaChanged() i tutaj w Gui byśmy wykorzystali to zdarzenie i przypisywali do property text zmienioną wartość :). Póki co mogłoby to tak wyglądać , w przyszłości można pomyśleć jak to sparametryzować żeby był jeden event do wszystkich właściwości.